docs/docs/references/groups.mdx

```diff
@@ -8,6 +8,12 @@ To view and manage your groups, go to the `Organization Settings` > `Users & gro
 
 <img src={GroupsManagementPage} width="1230" height="520" style={{display: "block", margin: "0 auto 20px auto"}}/>.
 
+:::info
```

Comment:
in another PR let's move this to pills at the top of the page

Reply:
done in this PR to resolve conflict with this PR https://github.com/lightdash/lightdash/pull/8727#issuecomment-1910263460

doc preview: https://deploy-preview-8704--peaceful-bassi-cbf284.netlify.app/references/groups